lstchain/io/tests/test_config.py

```diff
@@ -4,6 +4,9 @@
 def test_get_standard_config():
     config.get_standard_config()
 
+def test_get_srcdep_config():
+    config.get_srcdep_config()
```

Comment:
Please add a little check to make sure what this function returns is actually what you expect.